### PR TITLE
Fix Rerun 3.9 compatibility: Remove imports of `TypeAlias`

### PR DESCRIPTION
**Pull request by emilk**
### What
* This should fix https://github.com/rerun-io/rerun/issues/7793

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by emilk** [comment]
@rerun-bot full-check

File: rerun_py/rerun_bindings/__init__.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, TypeAlias, Union
```
Comment (emilk) [inline rerun_py/rerun_bindings/__init__.py:3]: I believe these were just unnecessary??

Comment (jleibs) [reply rerun_py/rerun_bindings/__init__.py:3]: Oops, yeah. Why didn't Mypy warn about those?

Comment (jleibs) [reply rerun_py/rerun_bindings/__init__.py:3]: Oops, yeah. Why didn't Mypy warn about those?

**Review by jleibs — APPROVED** [review]

**Comment by jleibs** [comment]
Still showing issues:
```
>>> import rerun
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jleibs/py38/venv/lib/python3.8/site-packages/rerun_sdk/rerun/__init__.py", line 20, in <module>
    from . import (
  File "/home/jleibs/py38/venv/lib/python3.8/site-packages/rerun_sdk/rerun/dataframe.py", line 15, in <module>
    from rerun_bindings.types import (
  File "/home/jleibs/py38/venv/lib/python3.8/site-packages/rerun_bindings/types.py", line 37, in <module>
    ComponentLike: TypeAlias = Union[str, type["ComponentMixin"]]
TypeError: 'type' object is not subscriptable
```

**Comment by jleibs** [comment]
@rerun-bot full-check

**Comment by jleibs** [comment]
@rerun-bot full-check

**Pull request merged**